integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/KnnPredicateSpecificsIT.java

```diff
@@ -1036,13 +1033,66 @@ public static List<? extends Arguments> similarityFilterFloat() {
 			return args;
 		}
 
+		@ParameterizedTest
+		@MethodSource
+		void similarityFilterFloatOpenSearch(SimpleMappedIndex<IndexBinding> index, float similarity, float score,
```

Comment:
I couldn't get OpenSearch to behave the same as Lucene and Elasticsearch for this similarity/distance filter, so I've just added a different test for it instead...